assets/lucashifacts

```diff
@@ -100,3 +100,11 @@ lucashi walked into the fridge door and spilled vegetables all over himself
 lucashi once dated an ex-convict
 lucashi is a fully licensed motorcycle rider in Ontario
 lucashi has climbed to the peak of mt. Elbert the tallest moutain in the rocky mountains
+the only bone lucashi has ever broken is his wrist bone
+lucashi has trained new employees at CoreAVI
+lucashi has a tendency to have drinks poured all over his pant
+lucashi got so drunk at a work party once, he forgot his underwear in his coworkers car
+lucashi survived the tragic 5.0 magnitude earthquake that struck Ottawa in the summer of 2010
+lucashi at the request of his CEO, took shots with many corporate executives
```

Comment:
comma after lucashi? Or mdashes

Reply:
So like "lucashi, at the request of his CEO, took ..." ?